src/android/Mapbox.java

```diff
@@ -168,7 +183,23 @@ public void run() {
             FrameLayout.LayoutParams params = new FrameLayout.LayoutParams(webViewWidth - left - right, webViewHeight - top - bottom);
             params.setMargins(left, top, right, bottom);
             mapView.setLayoutParams(params);
-
+			//@anothar we have to override back button or app will just close
+            mapView.setOnKeyListener(new View.OnKeyListener() {
```

Comment:
Hi @pke it is not error. But you are right is doesn't relate to image markers. Is fixes bug [issue 26](https://github.com/Telerik-Verified-Plugins/Mapbox/issues/26). To be more exact without this fix android plugin closes your app when you press back button on phone. I know it is not good to add this fix to this PR. But I wasn't sure when/if everything gets merged and needed this fix.